themes/src/main/resources/theme/keycloak-preview/account/messages/messages_en.properties

```
@@ -74,6 +74,8 @@ webauthn-display-name=Security Key
 webauthn-help-text=Use your security key to log in.
 webauthn-passwordless-display-name=Security Key
 webauthn-passwordless-help-text=Use your security key for passwordless log in.
+basic-authentication=Basic Authentication
+basic-auth-help-text=Signin with username and password.
```

Just a small nitpick. Of course I'm not a native speaker but "Signin" doesn't seem to be correct. Maybe it's some slang but even Google seems to be sure it's not used anywhere.

Shouldn't it be like this?
```suggestion
basic-auth-help-text=Sign in with username and password.
```

---

I think you are right.  People use "Login", but I don't see "Signin".